packages/salesforcedx-vscode-lwc/resources/static/js-meta.xsd

```diff
@@ -0,0 +1,291 @@
+          </xs:documentation>
+    </xs:annotation>
+    <xs:attribute type="xs:string" name="datasource" use="optional" />
+    <xs:attribute type="xs:string" name="default" use="optional" />
+    <xs:attribute type="xs:string" name="description" use="optional" />
+    <xs:attribute type="xs:string" name="label" use="optional" />
+    <xs:attribute type="xs:integer" name="max" use="optional" />
+    <xs:attribute type="xs:integer" name="min" use="optional" />
+    <xs:attribute type="xs:string" name="name" use="required" />
+    <xs:attribute type="xs:string" name="placeholder" use="optional" />
+    <xs:attribute type="xs:string" name="required" use="optional" />
+    <xs:attribute type="xs:string" name="role" use="optional" />
+    <xs:attribute type="xs:string" name="type" use="required" />
+  </xs:complexType>
+
+  <!-- supportedFormFactors SXD-->
+  <xs:complexType name="supportedFormFactors">
+    <xs:sequence>
+      <xs:element name="supportedFormFactor" maxOccurs="unbounded" minOccurs="1">
+        <xs:complexType>
+          <xs:annotation>
+            <xs:documentation>
+            Defines which form factor or device the component supports. Use one supportedFormFactor tag for each supported form factor.
+            </xs:documentation>
+          </xs:annotation>
+          <xs:attribute type="supportedFormFactortype" name="type" use="required" />
+        </xs:complexType>
+
+
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:simpleType name="supportedFormFactortype">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Large" />
+      <xs:enumeration value="Small" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <!-- targetConfigs SXD-->
+  <xs:complexType name="targetConfigs">
+    <xs:sequence>
+      <xs:element name="targetConfig" maxOccurs="unbounded" minOccurs="0">
+        <xs:complexType>
+          <xs:annotation>
+            <xs:documentation>
+                            Use a separate targetConfig for each different page type configuration.
+                      </xs:documentation>
+          </xs:annotation>
+
+          <xs:all>
+            <xs:element type="objects" name="objects" minOccurs="0" />
+            <xs:element type="property" name="property" minOccurs="0" />
+            <xs:element type="supportedFormFactors" name="supportedFormFactors" minOccurs="0" />
+          </xs:all>
+
+          <xs:attribute type="xs:string" name="configurationEditor" use="optional">
+            <xs:annotation>
+              <xs:documentation>
+                                Registers a custom property editor for a flow screen component. A custom property editor is a Lightning web component that provides a custom UI for configuring a custom flow screen component’s input attributes. See Develop Custom Property Editors for Flow Screen Components (Beta).
```

Comment:
Delete "See Develop Custom Property Editors for Flow Screen Components (Beta)" because that sentence will be hard to maintain when this moves out of Beta and it's not clear where this doc is. (I know that it's in the Lightning Web Components Developer Guide but that won't be obvious in VisualStudio.